packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -529,6 +529,13 @@ $Calendar-dayPicker-margin: 10px;
       @include ms-right(13px);
     }
   }
+
+  //when month picker is only visible
+  .monthPickerOnly{
+    .wrap{
+      padding: 10px 7px;
+    }
+  }
```

Comment:
why do we need special styling for this case?

Reply:
This fixes the scenario from the screenshot above (month picker only).  It isn't needed when there is an overlayed month picker / date picker (prop:  showMonthPickerAsOverlay), because the header for the date picker and month picker are already in the same spot.